modules/order/config/install/profile.type.customer.yml

```diff
@@ -9,3 +9,4 @@ label: Customer
 registration: false
 multiple: true
 weight: 0
+use_revisions: true
```

Comment:
What about existing sites? We probably need an update function.

Reply:
Fixed.